app/Controller/PagesController.php

```diff
@@ -38,6 +38,45 @@ class PagesController extends AppController {
 	public $uses = array();
 
 /**
+ * TooManyMethods
+*/
+public function display1() {
+}
+public function display2() {
+}
+public function display3() {
+}
+public function display4() {
+}
+public function display5() {
+}
+public function display6() {
+}
+public function display7() {
+}
+public function display8() {
+}
+public function display9() {
+}
+public function display10() {
+}
+public function display11() {
+}
+public function display12() {
+	// empty line for test commit
```

Comment:
[staging] [code_sniffer] Spaces must be used to indent lines; tabs are not allowed